acceptance_tests/features/pages/reporting_unit.py

```diff
@@ -0,0 +1,14 @@
+from acceptance_tests import browser
+from config import Config
+
+
+def go_to(ru_ref):
+    browser.visit(f"{Config.RESPONSE_OPERATIONS_UI}/reporting-units/{ru_ref}")
```

Comment:
Can we keep these consistent across the repo please?

Reply:
I had a look into it and f-strings are actually a faster method of making strings like this and they're more readable. 
I'm happy to change it back but in the longer run I think this is the better way to go

Reply:
I'd agree that f-strings are actually more readable and I think as a team we've adopted f-strings in the majority of other repositories. Two ways of changing it are raise a tech debt to change it all or change it as we go